crates/daphne-service-utils/src/durable_requests/bindings/aggregate_store.rs

```diff
@@ -47,6 +47,14 @@ fn durable_name_task(version: DapVersion, task_id_hex: &str) -> String {
 pub struct AggregateStoreMergeReq {
     pub contained_reports: Vec<ReportId>,
     pub agg_share_delta: DapAggregateShare,
+    pub options: AggregateStoreMergeOptions,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+// TODO(mendess): delete. Only here to suport `legacy_req_parse`
+#[derive(serde::Deserialize)]
+pub struct AggregateStoreMergeOptions {
+    pub skip_replay_protection: bool,
```

Comment:
It's notable that skipping replay protection degrades privacy by allowing a malicious co-Aggregator to replay reports. I think this needs to be reflected in how we document this feature. We should note (1) privacy is degraded and (2) it's intended to be used in an incident and needs to be re-enabled after the incident is mitigated.